requirements/build-requirements.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash 
+SCRIPT=$(readlink -f "$0")
+REQPATH=$(dirname $SCRIPT)
```

Comment:
I guess the `REQPATH` here should be relative to the root directory of the repository always, to avoid adding personal paths as part of the comments auto-generated by `pip-compile` (see comment below)

Reply:
If you prefer I can do this exactly the way you said (making the REQPATH relative to the root). The way I currently left it in the PR allows the script to be run from the root directory or from the requirements directory, without leaving an absolute path in the requirements files.